### ISSUE TITLE
About the hyperparameters of large models 

### ISSUE THREAD

**Issue by yuchen3890** [ZhengxiangShi/DePT#10]
Hi, I would like to know the experiment details about training large models like T5-3B, llama-2-7B/13B. Will the learning rate and batch size be the same as training T5-base? Also, what will the design of prompt and Lora be (the number of PREFIX_LENGTH, R)? Is it convenient for you to provide the training shell script files for reference? Thank you very much.

**Comment by HungerPWAY** [comment]
I have the same concern regarding the parameters of large models. A single 3090 GPU is insufficient to support the use of a large model with the same parameters of T5-base.

**Comment by HungerPWAY** [comment]
I have the same concern regarding the hyper-parameters of large models. A single 3090 GPU is insufficient to support the use of a large model with the same hyper-parameters of T5-base.
